dataplicity/m2m/wsclient.py

```diff
@@ -262,6 +286,20 @@ def open_channels(self):
         """List of open channels."""
         return self.channels.keys()
 
+    @property
+    def time_since_last_packet(self):
+        """Time, in seconds, since the last packet."""
+        return time.time() - self.last_packet_time
+
+    @property
+    def is_responding(self):
```

Comment:
This is polled every 15 seconds, and will reconnect if the server is not responding (or closed).